supply/lib/supply/setup.rb

```diff
@@ -44,7 +44,7 @@ def download_images(listing)
       IMAGES_TYPES.each do |image_type|
         if ['featureGraphic'].include?(image_type)
           # we don't get all files in full resolution :(
-          UI.message("Due to a limitation of the Google Play API, there is no way for `supply` to download your existing feature graphics. Please copy your feature graphics into `metadata/android/en-US/images/`")
+          UI.message("Due to a limitation of the Google Play API, there is no way for `supply` to download your existing feature graphic. Please copy your feature graphic to `metadata/android/en-US/images/featureGraphic.png`")
```

Comment:
The `en-US` only matches sometimes here and should probably be dynamified in a future PR, right?

Reply:
Seems like yes - though I think it's fairly clear that if I have other supported lang then I'd put it in a different folder.

Reply:
Oh, I wrote an issue about that in the past: https://github.com/fastlane/fastlane/issues/9962 ;)